test/server/listener_manager_impl_quic_only_test.cc

```diff
@@ -172,6 +172,132 @@ TEST_P(ListenerManagerImplQuicOnlyTest, QuicListenerFactoryAndSslContext) {
   EXPECT_CALL(*listener_factory_.socket_, close()).Times(0u);
   EXPECT_TRUE(listener_factory_.socket_->socket_is_open_);
 }
+
+TEST_P(ListenerManagerImplQuicOnlyTest, QuicWriterFromConfig) {
```

Comment:
Is there a test to test that quic uses GSO if the writer config is absent?

Reply:
Yes, the previous test `QuicListenerFactoryAndSslContext` verifies this:

`  EXPECT_EQ(udp_packet_writer->isBatchMode(), Api::OsSysCallsSingleton::get().supportsUdpGso());
`

Reply:
would it be possible have a default config stashed somewhere and only update the fields we want to update for these tests?

Reply:
Hm. So the new test I added adds a few fields to the config in the previous test. So I added a method to return the common bit of config between them, and just append the new bits in the new test. So maybe this is now done?

But the other tests seem to use relatively different configs in ways that don't entirely make it obvious quite how to share them... Unless perhaps you were thinking that instead of creating the config via strings it should be via objects?